apps/judicial-system/backend/src/app/modules/notification/test/internalNotificationController/sendRulingNotifications.spec.ts

```diff
@@ -205,7 +205,7 @@ describe('InternalNotificationController - Send ruling notifications', () => {
             ],
             cc: mockConfig.email.prisonEmail.split(',').slice(1),
 
-            subject: 'Úrskurður um gæsluvarðhald',
+            subject: 'Úrskurður í máli ' + theCase.courtCaseNumber,
```

Comment:
Consider using template literal string here to keep consistency with the other subjects below